OneCommunityGlobal/HighestGoodNetworkApp
# Leo Add ability to create a team from the “Add Team” option of User Management

# Description
![Screenshot 2023-07-12 001931](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/122568562/df43efea-6405-4641-b02f-5e96eaed2101)
…
## Related PRS (if any):
This frontend PR is related to [#453](https://github.com/OneCommunityGlobal/HGNRest/pull/453) backend PR
…

## Main changes explained:
- Update AddTeamPopup.jsx for including new "create new team" button
- Able to create a new team in the add team popup
…


## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin user
4. Admin/Owner Login → Other Links → User Management → Create New User → Assign Team 
5. Check if the create new team button is able to create a new team and add team immediately

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/122568562/78fb1346-b78d-406d-b387-56e77b42e5ec




## Note:
Include the information the reviewers need to know.
